model/riscv_addr_checks.sail

```diff
@@ -77,14 +76,13 @@ type ext_fetch_addr_error = unit
  * returns:  the *virtual* memory address to use for the fetch.
  *           any address translation errors are reported for pc, not the returned value.
  */
-function ext_fetch_check_pc(start_pc : xlenbits, pc : xlenbits) -> Ext_FetchAddr_Check(ext_fetch_addr_error) =
-  Ext_FetchAddr_OK(pc)
+function ext_fetch_check_pc(start_pc : xlenbits, pc : xlenbits) -> Ext_FetchAddr_Check(ext_fetch_addr_error) = Ext_FetchAddr_OK(
+  pc,
+)
```

Comment:
This is tricky because it comes down to whether you allow
```
f x = <expression>
```
or more aggressively format as
```
f x =
  <expression>
```
even when either `f x` is short or `<expression>` would be forced onto multiple lines anyway.